python/cucim/src/cucim/skimage/feature/corner.py

```diff
@@ -223,7 +228,7 @@ def _hessian_matrix_with_gaussian(image, sigma=1, mode='reflect', cval=0,
 
     # 2.) apply the derivative along another axis as well
     axes = range(ndim)
-    if order == 'rc':
+    if order == 'xy':
```

Comment:
yes, it was a bug.

In practice, the actual order does not matter for internal functions like `skimage.filters.frangi`, `skimage.feature.blob_doh`, or `skimage.feature.shape_index` that calls this function since these rely only on the eigenvalues or determinant of the matrix which are fortunately the same for either order.

Reply:
The change here makes things consistent with `skimage.feature.structure_tensor` which also uses the same logic, but had the ordering correct.